<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1210</width>
    <height>800</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(32, 71, 105);
border-color: rgb(255, 255, 255);</string>
  </property>
  <widget class="QLabel" name="word_line">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>40</y>
     <width>261</width>
     <height>55</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 25pt &quot;Yrsa&quot;;</string>
   </property>
   <property name="text">
    <string>WORD</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="shuffle_button">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>720</y>
     <width>141</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;}
QPushButton:hover {background-color: rgb(20, 75, 130);}</string>
   </property>
   <property name="text">
    <string>shuffle</string>
   </property>
  </widget>
  <widget class="QPushButton" name="next_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>570</y>
     <width>281</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;}
QPushButton:hover {background-color: rgb(20, 75, 130);}</string>
   </property>
   <property name="text">
    <string>next</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pronunciation_UK">
   <property name="geometry">
    <rect>
     <x>660</x>
     <y>120</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
selection-background-color: rgb(30, 85, 138);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../../../../.designer/backup/WindowsGraphics</normalon>
     <activeon>:/voiceButtonIcon.png/voiceButtonIcon.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QLabel" name="pronunciation_UK_text">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>230</y>
     <width>61</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 25pt &quot;Yrsa&quot;;</string>
   </property>
   <property name="text">
    <string>UK</string>
   </property>
  </widget>
  <widget class="QLabel" name="pronunciation_US_text">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>230</y>
     <width>51</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 25pt &quot;Yrsa&quot;;</string>
   </property>
   <property name="text">
    <string>US</string>
   </property>
  </widget>
  <widget class="QPushButton" name="pronunciation_US">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>120</y>
     <width>90</width>
     <height>90</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: white;
selection-background-color: rgb(30, 85, 138);</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset>
     <normalon>../../../../.designer/backup/WindowsGraphics</normalon>
     <activeon>:/voiceButtonIcon.png/voiceButtonIcon.png</activeon>
    </iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>40</width>
     <height>40</height>
    </size>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="exit_button">
   <property name="geometry">
    <rect>
     <x>45</x>
     <y>720</y>
     <width>91</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;}
QPushButton:hover {background-color: rgb(20, 75, 130);}</string>
   </property>
   <property name="text">
    <string>exit</string>
   </property>
  </widget>
  <widget class="QPushButton" name="right_answer_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>650</y>
     <width>281</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;}
QPushButton:hover {background-color: rgb(20, 75, 130);}</string>
   </property>
   <property name="text">
    <string>answer</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="input_text">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>370</y>
     <width>281</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:rgb(200, 211, 223);
font: 19pt &quot;Yrsa&quot;;
color:black;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;
selection-color: rgb(101, 145, 232);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QPushButton" name="submit_button">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>450</y>
     <width>281</width>
     <height>60</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton {background-color:rgb(30, 85, 138);
font: 19pt &quot;Yrsa&quot;;
color:white;
selection-background-color: rgb(255, 255, 255);
border-style: outset;
border-width: 1px;
border-radius: 15px;
border-color: black;
padding: 4px;}
QPushButton:hover {background-color: rgb(20, 75, 130);}</string>
   </property>
   <property name="text">
    <string>submit</string>
   </property>
  </widget>
  <widget class="QLabel" name="wrong_answer_line">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>300</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 23pt &quot;Yrsa&quot;;</string>
   </property>
   <property name="text">
    <string>Wrong answer</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
